README.md

```diff
@@ -96,9 +105,37 @@ Each lion should have:
 
 Create a new lion instance with the name `simba`
 
-```ruby
-# code here
-```
+
+class Animal
+attr_accessor :name
+
+def initialize name
+@name = name
+end
+def greet
+    p "Hello, my name is #{@name}!"
+    end
+    Pumba = Animal.new("Pumba")
```

Comment:
You want to create an instance of the class **outside** of the class. Also, only class names should have the first letter capitalized - not instances.